### PR TITLE
chore(dashboard): Apply field badge to current session tags

### PR DESCRIPTION
**Pull request by shruthilayaj**
Updated the tag badge

Before:
Screen Shot 2022-05-30 at 9 14 26 AM

After:
Screen Shot 2022-05-30 at 9 16 22 AM

### CODE REVIEW

File: static/app/views/dashboardsV2/widgetBuilder/releaseWidget/fields.tsx [hunk 1]
```diff
@@ -31,8 +31,6 @@ export const SESSIONS_OPERATIONS: Readonly<
   Record<SessionsOperation, SessionAggregationColumn>
 > = {
   sum: {
-    columnTypes: ['integer'],
```
Comment (shruthilayaj) [inline static/app/views/dashboardsV2/widgetBuilder/releaseWidget/fields.tsx:34]: unrelated code cleanup, removing some redundant code

**Review by narsaynorath — APPROVED** [review]
Screen Shot 2022-05-30 at 1 41 09 PM

I noticed the group by step still renders as a tag and not a field. The change should just be to change the `kind` when generating the options in `widgetBuilder/buildSteps/groupByStep/index.tsx`, line 21

**Comment by shruthilayaj** [comment]
> Screen Shot 2022-05-30 at 1 41 09 PM
> 
> I noticed the group by step still renders as a tag and not a field. The change should just be to change the `kind` when generating the options in `widgetBuilder/buildSteps/groupByStep/index.tsx`, line 21

fixed in 8f06746b5dbde5bf66967fea11831356293dea48
